server/notification-providers/feishu.js

```diff
@@ -25,51 +25,59 @@ class Feishu extends NotificationProvider {
 
             if (heartbeatJSON["status"] === DOWN) {
                 let downdata = {
-                    msg_type: "post",
-                    content: {
-                        post: {
-                            zh_cn: {
-                                title: "UptimeKuma Alert: [Down] " + monitorJSON["name"],
-                                content: [
-                                    [
-                                        {
-                                            tag: "text",
-                                            text:
-                                                "[Down] " +
-                                                heartbeatJSON["msg"] +
-                                                `\nTime (${heartbeatJSON["timezone"]}): ${heartbeatJSON["localDateTime"]}`
-                                        },
-                                    ],
-                                ],
+                    msg_type: "interactive",
+                    card: {
+                        config: {
+                            update_multi: true,
```

Comment:
Sorry, but the translation via google translate is not really making sense on this parameter.
Could you give context what this does?

![image](https://github.com/louislam/uptime-kuma/assets/26258709/10699353-6218-4a87-9c9e-6e9bc4cbc023)

Reply:
Sorry, my mistake, update_multi should be set to `false`

Reply:
This parameter is used to update the content of the message card sent by the application. In the monitor scenario, this is not necessary.